Makefile

```diff
@@ -38,6 +38,10 @@ kubeapps/tiller-proxy:
 	CGO_ENABLED=0 GOOS=linux go build -installsuffix cgo -o ./cmd/tiller-proxy/proxy-static ./cmd/tiller-proxy
 	docker build -t kubeapps/tiller-proxy:$(VERSION) -f cmd/tiller-proxy/Dockerfile cmd/tiller-proxy
 
+kubeapps/chartsvc:
```

Comment:
This change and the Dockerfile below are unrelated to the bug fix. It's just for building the image faster (2m vs 8s in my local machine)

Reply:
I do not think that this change should be done here and it should probably be part of a bigger discussion. Maybe as part of https://github.com/kubeapps/kubeapps/issues/461 or could you create an issue explaining what problem you are having?

Please do not add unrelated code to a patch still less if it has already been accepted.

Thanks

Reply:
had to try :P I'll remove this and open a different issue for that